crates/webidl/src/util.rs

```diff
@@ -285,3 +286,40 @@ pub fn create_setter(
         vec![backend::ast::BindgenAttr::Setter(Some(raw_ident(name)))],
     )
 }
+
+pub fn wrap_import_function(function: backend::ast::ImportFunction) -> backend::ast::Import {
+    backend::ast::Import {
+        module: None,
+        version: None,
+        js_namespace: None,
+        kind: backend::ast::ImportKind::Function(function),
+    }
+}
+
+/// ChromeOnly is for things that are only exposed to priveleged code in Firefox.
+pub fn is_chrome_only(ext_attrs: &[Box<ExtendedAttribute>]) -> bool {
+    ext_attrs.iter().any(|external_attribute| {
+        return match external_attribute {
+            box ExtendedAttribute::ArgumentList(al) => {
+                println!("ArgumentList");
+                al.name == "ChromeOnly"
+            },
+            box ExtendedAttribute::Identifier(i) => {
+                println!("Identifier");
+                i.lhs == "ChromeOnly"
+            },
+            box ExtendedAttribute::IdentifierList(il) => {
+                println!("IdentifierList");
+                il.lhs == "ChromeOnly"
+            },
+            box ExtendedAttribute::NamedArgumentList(nal) => {
+                println!("NamedArgumentList");
+                nal.lhs_name == "ChromeOnly"
+            },
+            box ExtendedAttribute::NoArguments(_na) => {
```

Comment:
We are missing a pattern here, which is why you didn't get any change needed for `Event.rs`, namely:
```rust
ExtendedAttribute::NoArguments(webidl::ast::Other::Identifier(name)) => name == "ChromeOnly",
```